### PR TITLE
rbd-mirror: replicate cloned images

### PR DESCRIPTION
**Pull request by dillaman**

### CODE REVIEW

**Comment by dillaman** [comment]
Requires fixed from #9360

**Comment by trociny** [comment]
@dillaman could you please rebase?

Also, add "Fixes: ..." to "rbd-mirror: add support ..." commit log?

File: qa/workunits/rbd/rbd_mirror.sh [hunk 1]
```diff
@@ -416,19 +425,32 @@ test_status_in_pool_dir()
 create_image()
 {
     local cluster=$1
-    local image=$2
+    local pool=$2
+    local image=$3
 
-    rbd --cluster ${cluster} -p ${POOL} create --size 128 \
+    rbd --cluster ${cluster} -p ${pool} create --size 128 \
```
Comment (trociny) [inline qa/workunits/rbd/rbd_mirror.sh]: I think we need to fix all other functions too that use the hardcoded ${POOL}? test_image_replay_state, wait_for_replay_..., test_status_in_pool_dir...

File: qa/workunits/rbd/rbd_mirror.sh [hunk 2]
```diff
@@ -219,94 +219,16 @@ admin_daemon()
     ceph --admin-daemon ${asok_file} $@
 }
 
-status()
```
Comment (trociny) [inline qa/workunits/rbd/rbd_mirror.sh]: Then also need to remove `status` command from the example section in the beginning of the script.

Comment (dillaman) [inline qa/workunits/rbd/rbd_mirror.sh]: I can put it back -- I saw that it wasn't in-use within the script so I yanked it.

Comment (trociny) [inline qa/workunits/rbd/rbd_mirror.sh]: It is useful (at least for me) when testing manually to check the current state, but sure I can live without it (especially if it adds additional work to maintain the script).

Comment (dillaman) [inline qa/workunits/rbd/rbd_mirror.sh]: No worries, I'll put it back and expand it to cover both pools.

**Comment by dillaman** [comment]
@trociny re-added the ```status``` function

**Comment by trociny** [comment]
lgtm
passed local testing
teuthology tests: http://pulpito.ceph.com/trociny-2016-05-31_22:06:08-rbd-wip-mgolub-testing---basic-vps/ (some jobs failed but nothing look related)

**Pull request merged**